packages/integration-tests/utils/generatePlugin.ts

```diff
@@ -99,6 +106,7 @@ class SentryScenarioGenerationPlugin {
             '@sentry/browser': 'Sentry',
             '@sentry/tracing': 'Sentry',
             '@sentry/integrations': 'Sentry.Integrations',
+            '@sentry/replay': 'Sentry.Integrations',
```

Comment:
Don't know much about Replay, but if it's under `Sentry` namespace, when used from CDN:

```suggestion
            '@sentry/replay': 'Sentry.Replay',
```

Reply:
It's under `Sentry.Integrations.Relay` when used from the CDN: 
https://docs.sentry.io/platforms/javascript/session-replay/#set-up